### PR TITLE
Update paths and ports for Dogecoin

### PR DESCRIPTION
**Pull request by IncognitoJam**
Changed the paths and ports referenced in the configuration to match dogecoind/the normal network ports. Also updated the `install.sh` script to point to this repository.

I tried to change as few things as possible, so left variable names and comments alone. The actual chaincfg and network params can come later 😄 

## Open questions

I wasn't sure whether to change paths like "bitcoin-data" to "dogecoin-data" which is the location where the docker volume is mounted.

### CODE REVIEW

**Comment by patricklodder** [comment]
> I wasn't sure whether to change paths like "bitcoin-data" to "dogecoin-data" which is the location where the docker volume is mounted.

I think this is a good change - as long as we reflect this also in `.*ignore` files

**Comment by IncognitoJam** [comment]
Turns out none of the code changes were having an affect because `main.go` was still importing from `coinbase/rosetta-bitcoin`... I tried swapping out only the imports for the files changed, but Go complains that there's a type mismatch
```
#19 36.37 # github.com/rosetta-dogecoin/rosetta-dogecoin
#19 36.37 ./main.go:93:30: cannot use cfg (type *"github.com/rosetta-dogecoin/rosetta-dogecoin/configuration".Configuration) as type *"github.com/coinbase/rosetta-bitcoin/configuration".Configuration in argument to indexer.Initialize
#19 36.37 ./main.go:93:30: cannot use client (type *"github.com/rosetta-dogecoin/rosetta-dogecoin/bitcoin".Client) as type indexer.Client in argument to indexer.Initialize:
#19 36.37       *"github.com/rosetta-dogecoin/rosetta-dogecoin/bitcoin".Client does not implement indexer.Client (wrong type for GetRawBlock method)
#19 36.37               have GetRawBlock(context.Context, *types.PartialBlockIdentifier) (*"github.com/rosetta-dogecoin/rosetta-dogecoin/bitcoin".Block, []string, error)
#19 36.37               want GetRawBlock(context.Context, *types.PartialBlockIdentifier) (*"github.com/coinbase/rosetta-bitcoin/bitcoin".Block, []string, error)
#19 36.37 ./main.go:166:40: cannot use cfg (type *"github.com/rosetta-dogecoin/rosetta-dogecoin/configuration".Configuration) as type *"github.com/coinbase/rosetta-bitcoin/configuration".Configuration in argument to services.NewBlockchainRouter
```
So I think we will have to swap all of the imports for `coinbase/rosetta-bitcoin` to our package

**Comment by michilumin** [comment]
So this looks sound to me, only commentary is making the comments reflect the actual method names (bitcoind -> dogecoind) but besides that, looks proper.

**Review by michilumin — APPROVED** [review]
Looks good, approved recent comment edits.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ name: CI
 on: [push, pull_request]
 
 env:
-  GO_PACKAGES: ./services/... ./indexer/... ./bitcoin/... ./configuration/...
+  GO_PACKAGES: ./services/... ./indexer/... ./bitcoin/... ./dogecoin/...
```
Comment (podnov) [inline .github/workflows/ci.yml:20]: `configuration` changed to `dogecoin` instead of `bitcoin` changing to `dogecoin`, is this intended?

Comment (podnov) [reply .github/workflows/ci.yml:20]: This same thing applies to `Makefile`.

**Pull request merged**